<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tea_querystu</class>
 <widget class="QDialog" name="tea_querystu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选课和成绩管理系统</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询学生成绩</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>141</y>
     <width>611</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5,1">
    <item>
     <widget class="QComboBox" name="cbb_querymode">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>按姓名查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按学号查询</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按班级查询</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_info"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_doquery">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>开始查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
